cloudigrade/account/views.py

```diff
@@ -1,10 +1,35 @@
 """DRF API views for the account app."""
+from django.utils.translation import gettext as _
 from rest_framework import exceptions, mixins, status, viewsets
 from rest_framework.response import Response
+from rest_framework.serializers import ValidationError
 
 from account import serializers
 from account.exceptions import InvalidCloudProviderError
-from account.models import Account, AwsAccount
+from account.models import (Account,
+                            AwsAccount,
+                            Instance)
+
+
+def convert_param_to_int(name, value):
+    """Check if a value is convertable to int.
+
+    :param name: The field name being validated
```

Comment:
Use Google style docstrings.

We may move to the RST style later, but consistency wins for now.

Reply:
Done.